deps/Makefile

```diff
@@ -30,7 +30,7 @@ CONFIGURE_COMMON += --host=$(XC_HOST)
 endif
 ifeq ($(OS),WINNT)
 ifneq ($(USEMSVC), 1)
-CONFIGURE_COMMON += LDFLAGS=-Wl,--stack,8388608
+CONFIGURE_COMMON += LDFLAGS="$(LDFLAGS) -Wl,--stack,8388608"
```

Comment:
with autoconf / make, i always wonder if this'll go through correctly, or if it'll later get stomped on by recursive make passing down the flag from the env. i'm never quite sure if / how we should set the recursion options: https://www.gnu.org/software/make/manual/html_node/Options_002fRecursion.html#Options_002fRecursion